src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -201,8 +200,8 @@ class DirectoryIterator::NativeIterator::Pimpl
     DIR* dir;
 };
 
-DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard)
-    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard))
+DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard_)
+    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard_))
```

Comment:
A slight grimace here.  I generally expect variables with a trailing `_` to be members.  Perhaps `wildCardIn` would not bring the wrath of the character counters?